.changeset/spotty-penguins-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/rating": minor
```

Comment:
I think this would be a major change, since it's relying on the addition and usage of a new class.